### PR TITLE
copr: table scan ignores point get while encounter common handle

### PR DESCRIPTION
**Pull request by iosmanthus**
Signed-off-by: Iosmanthus Teng <myosmanthustree@gmail.com>

### What problem does this PR solve?
`BatchTableScanExecutor` ignores point get while encounter common handles keys.

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

* `BatchTableScanExecutor` ignores point get while encountering common handle keys.

### CODE REVIEW

File: components/tidb_query_vec_executors/src/table_scan_executor.rs [hunk 1]
```diff
@@ -49,7 +49,7 @@ impl<S: Storage> BatchTableScanExecutor<S> {
         let mut columns_default_value = Vec::with_capacity(columns_info.len());
         let mut column_id_index = HashMap::default();
 
-        let primary_column_ids_set = primary_column_ids.iter().collect::<HashSet<_>>();
+        let primary_column_ids_set = primary_column_ids.iter().cloned().collect::<HashSet<_>>();
```
Comment (breeswish) [inline components/tidb_query_vec_executors/src/table_scan_executor.rs:52]: This clone can be avoided.

**Comment by coocood** [comment]
LGTM

**Comment by TennyZhuang** [comment]
LGTM

**Pull request merged**